NBitcoin.Tests/script_tests.cs

```diff
@@ -404,18 +404,20 @@ private void EnsureHasLibConsensus()
 			}
 			else if(RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
 			{
-				libConsensusDll = "libbitcoinconsensus.so.0.0.0";
+				libConsensusDll = "libbitcoinconsensus.so";
 			}
 			else
 			{
 				throw new NotSupportedException("Unknown operating system");
 			}
 
 			var bitcoinBinFolderPath = Path.GetDirectoryName("TestData");
-			var libConsensusPath = Path.Combine(bitcoinPath, "../../lib", libConsensusDll);
+			var libConsensusPath = Path.Combine(bitcoinPath, "../../lib/", libConsensusDll);

```

Comment:
The slash should not matter.

Reply:
i know. the problem is that everything works okay locally and that why i have to test silly things here.